src/main/webapp/app/course/competencies/forms/competency/competency-form.component.ts

```diff
@@ -25,14 +25,13 @@ export class CompetencyFormComponent extends CourseCompetencyFormComponent imple
         taxonomy: undefined,
         masteryThreshold: undefined,
         optional: false,
-        connectedLectureUnits: undefined,
     };
+    @Input()
+    competency: Competency;
```

Comment:
Sadly the formData above cannot be easily be converted to the new signal input, since it is also referenced in the super class with a more generic type. 
Since this would be more effort to refactor this component, I will not do it in this PR.